.evergreen/config.yml

```diff
@@ -62,6 +62,26 @@ functions:
         file: src/expansion.yml
 
   "install swift":
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
```

Comment:
I copied over the logic from the driver for dynamically calculating the latest patch version for Swift minor versions